### DISCUSSION TITLE
"Please first select some text to transform into a link." tooltip is shown even when there is text selected.

### DISCUSSION THREAD

**Discussion by harkairt** [singerdmx/flutter-quill#843] [Web] [answered]
Also, the button is clickable even without having anything selected. (And I think it should be as well)

Is it possible to hide this hint altogether?

**Comment by singerdmx** [comment, marked as answer]
Fixed in https://github.com/singerdmx/flutter-quill/commit/81720861b2c664b24301bb46522764c55a2e4ade
